xarray/tests/test_variable.py

```diff
@@ -2345,12 +2335,36 @@ def test_dask_rolling(self, dim, window, center):
         assert actual.shape == expected.shape
         assert_equal(actual, expected)
 
-    @pytest.mark.xfail(
-        reason="https://github.com/pydata/xarray/issues/6209#issuecomment-1025116203"
-    )
     def test_multiindex(self):
         super().test_multiindex()
 
+    @pytest.mark.skip
```

Comment:
Why add a new test and then skip it?
Also, add a reason why it is skipped

Reply:
```suggestion
```

The `@pytest.mark.skip` should not be there - thanks for noticing.

It's not new tests but instead of skipping it in the parent class we skip it specifically for `TestVariableWithDask`